pkg/common/common.go

```diff
@@ -21,6 +21,9 @@ const (
 	//#nosec G101
 	grafanaAgentExtraSecretName = "grafana-agent-secret"
 
+	AlloyLogAgentName      = "alloy-logs"
```

Comment:
This value is used for both the `--logging-agent` value and the Alloy app name in observability bundle.